xbmc/video/VideoDatabase.cpp

```diff
@@ -9008,10 +9008,8 @@ bool CVideoDatabase::GetMoviesByWhere(const std::string& strBaseDir, const Filte
           }
         }
 
-        path = itemUrl.ToString();
-
-        item->SetPath(path);
-        item->SetDynPath(movie.m_strFileNameAndPath);
+        item->SetPath(itemUrl.ToString());
+        item->SetDynPath(std::move(movie.m_strFileNameAndPath));
```

Comment:
no! this belongs to the video infotag and must remain valid.

Reply:
mmm yes I read too quickly, it's not a tag pointer as usual and the CFileItem constructor takes a const ref and proceeds to make a copy. So it's safe. I panicked a bit after seeing garbear's approval and impeding merge, sorry!